test/unit/item_test.rb

```diff
@@ -18,22 +18,22 @@ class ItemTest < ActiveSupport::TestCase
   test 'should save item when amount one percent is lesser than amount' do
     #given
     item = items(:one)
-    item.amount = 2
+    item.amount = 30
```

Comment:
```suggestion
    item.amount = 38
```
Bez poprawki wywala taki błąd:
```
Error:
ItemTest#test_should_save_item_when_amount_one_percent_is_lesser_than_amount:
ActiveRecord::RecordInvalid: Negatywne sprawdzenie poprawności: Items Suma z dotacji (38) musi mieścić się w wartości wpisu (30.0)
    test/unit/item_test.rb:27:in `block in <class:ItemTest>'
```